robot_dreams/Robot_dreams_3.0/Homework_27/script.js

```diff
@@ -0,0 +1,162 @@
+
+let a = new Array(13, 22, 31, 4, 5, 16, 7, 8, 9, 10);
+
+
+function addition(a, b) {
+  if (typeof a === "number" || typeof b === "number") {
+    return a + b;
+  } else {
+    console.log("Not a number");
```

Comment:
if your HW is about move result of function to HTML, here also should worth to show the error for user in HTML not in console